cmd/node/main.go

```diff
@@ -461,3 +418,33 @@ func initialState(
 	}
 	return state, transitions, nil
 }
+
+func createPriviServer(nodeConfig *config.NodeConfig) *privi.Server {
+	// TODO: read from persisted state about permissions.
+	policiesDirPath := nodeConfig.PermissionPolicyFilesDir()
+	_, err := os.ReadDir(policiesDirPath)
+	if errors.Is(err, os.ErrNotExist) {
+		log.Info().Msgf("Creating a policies dir path at %s", policiesDirPath)
+		err := os.Mkdir(policiesDirPath, 0o700)
+		if err != nil {
+			log.Fatal().
+				Err(err).
+				Msgf("error creating permission policies dir at %s", policiesDirPath)
+		}
+	} else if err != nil {
+		log.Fatal().Err(err).Msg("error reading from permission policies dir")
+	}
+
+	perms, err := permissions.NewStore(
+		fileadapter.NewAdapter(path.Join(policiesDirPath, "policies.csv")),
+		true,
+	)
+	if err != nil {
+		log.Fatal().Err(err).Msg("error creating permission store")
+	}
+
+	return privi.NewServer(
+		[]syntax.DID{},
```

Comment:
shouldn't this take in a non empty list? otherwise we should get rid of this and just read off all the dids in the file? i.e. if you're served by this privi your did is in there somewhere?